README.md

```diff
@@ -54,7 +54,7 @@ For old versions of git, that do not support this parameter, you can do:
     git submodule init
     git submodule update
 
-## Running QLever inside `docker`
+## Running QLever inside a `docker` container
```

Comment:
In _How to use_ the two variants are marked with **a** and **b**, so one could use that here and remove the **a** and **b** in the subsections.